app/assets/stylesheets/application.scss

```diff
@@ -15,3 +15,40 @@
  *= require foundation_and_overrides
 
  */
+
+ .full-width {
+   width: 100%;
+   margin: 0 auto;
+   max-width: initial;
+}
+
+.top-bar-section ul li a {
+  font-size: 4vmin;
+}
+
+/* MEDIA QUERIES
+==============================*/
+
+@media (min-width: 40.063em) {
+.distillery {
+  display: table-cell;
+}
+}
+
+@media (min-width: 40.063em) and (max-width: 64em) {
+.top-bar-section ul li a {
+  font-size: 3vmin;
+}
+}
+
+@media (min-width: 64.063em) and (max-width: 90em) {
+.top-bar-section ul li a {
+  font-size: 2.5vmin;
+}
+}
+
+@media (min-width: 90.063em) {
+.top-bar-section ul li a {
```

Comment:
Line should be indented 2 spaces, but was indented 0 spaces<br>Selector should have depth of applicability no greater than 2, but was 4